src/App.js

```diff
@@ -4,19 +4,40 @@ import Nav from './Nav'
 import MovieContainer from './MovieContainer'
 import MovieDetails from './MovieDetails'
 import './App.css';
-import movieData from './movieData';
-import movieDetails from './movieDetailData';
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieList: movieData.movies,
+      movieList: [],
       currentMovie: ""
     }
   }
 
+  componentDidMount = () => {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+      .then(response => {
+        if (!response.ok) {
+          throw Error()
+        } else {
+          return response.json()
+        }
+      })
+      .then(data => this.setState({movieList: data.movies}))
+      .catch(err => console.log(err))
+  }
+
+
   chooseMovie = (id) => {
-    this.setState({ currentMovie: movieDetails.movie })
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+      .then(response => {
+        if (!response.ok) {
+          throw Error()
+        } else {
+          return response.json()
+        }
+      })
+      .then(data => this.setState({currentMovie: data.movie}))
+      .catch(err => this.setState({currentMovie: "error"}))
```

Comment:
We should ask if we should move our fetch calls to their own file and then call them in App? Similar to what we've done in the past. Not sure what the "proper" way is in React.

Reply:
Yes!